pkg/cli/workflow/list_runs.go

```diff
@@ -34,6 +36,11 @@ func formatRunDuration(object interface{}, column string, field interface{}) str
 
 func formatRunStatus(object interface{}, column string, field interface{}) string {
 	if wr, ok := object.(*workflow.WorkflowRun); ok {
+		// The status may also contains the reason for a failed run, color the status only.
```

Comment:
Typo: _may contain_

Also, could you add an example of such status so it is clear why are you processing the string this way?

Reply:
done.